apps/catalogue/src/store/query/subcohortDetails.js

```diff
@@ -11,7 +11,10 @@ export default gql`
         name
       }
       name
-      description
```

Comment:
could we have the an optional 'description' field that return a default lang = 'en' value ? adding locale and value to each field lead to a lot of extra work and complexity. 

why is 'description' now 'descriptions' ( multiple descriptions ? )

Reply:
And it is 'descriptions' because that you have potential in multiple language (same for labels). But don't mind to change that back. It was just a great help to change this field name 
to find all the errors ;-)

Reply:
Can we move this to separate discussion?